src/main.py

```diff
@@ -198,12 +204,14 @@ def build(self):
 
         # Initialise settings object
         sett = settings_manager.Settings(sm)
+        self.settings_manager = sett
 
         # Initialise 'j'ob 'd'ata object
         jd = job_data.JobData(localization=self.l, settings_manager=sett)
 
         # Initialise 'm'achine object
         m = router_machine.RouterMachine(Cmport, sm, sett, self.l, jd)
+        self.machine = m
```

Comment:
same here for m.

Reply:
It was quicker than renaming all references to self.machine 🤣